resources/views/global/_latest-news-slider.antlers.html

```diff
@@ -0,0 +1,32 @@
+
+<div class="global-news-slider relative py-90 bg-concrete">
+  <div class="swiper overflow-visible container">
+    <div class="flex justify-between">
+      {{ if content.heading }}
+      <h3 class="heading">{{content.heading}}</h3>
+      {{ /if }}
+      <div class="xl-max:absolute bottom-0 right-20">
+        {{ partial:global/btn/arrow-left class="btn-arrow--garden btn-arrow--round swiper-arrow-left"}}
+        {{ partial:global/btn/arrow-right class="btn-arrow--garden btn-arrow--round swiper-arrow-right"}}
+      </div>
+    </div>
+
+    <div class="swiper-wrapper flex justify-between py-30">
+      {{ collection from="news" limit="{ content.number_of_articles }" }}
+          <div class="column swiper-slide w-full md:w-1/2 xl:w-1/3 relative">
```

Comment:
I know it doesn't make any style changes, however we like to define our widths as a fraction of 12 (columns in the grid), when we can, so we can quickly understand how the layout of the component is.
```suggestion
          <div class="column swiper-slide w-full md:w-6/12 xl:w-4/12 relative">
```